examples/PSScriptAnalyzerSettings.psd1

```diff
@@ -1,6 +1,5 @@
-# The PowerShell Script Analyzer will generate a warning
-# diagnostic record for this file due to a bug -
-# https://github.com/PowerShell/PSScriptAnalyzer/issues/472
+# For more information on PSScriptAnalyzer settings see:
+# https://github.com/PowerShell/PSScriptAnalyzer/blob/master/README.md#settings-support-in-scriptanalyzer
```

Comment:
Also linking to the shipped settings files could be useful: https://github.com/PowerShell/PSScriptAnalyzer/tree/master/Engine/Settings